src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -322,11 +322,14 @@ class WelcomePage extends Disposable {
 	}
 
 	private addVideoImageSource() {
-		const videoIntroduction = document.querySelector('#video_introduction') as HTMLImageElement;
-		const videoOverview = document.querySelector('#video_overview') as HTMLImageElement;
-
-		videoIntroduction.src = require.toUrl('./../../media/video_introduction.png');
-		videoOverview.src = require.toUrl('./../../media/video_overview.png');
+		const videoIntroduction = document.querySelectorAll('.video_introduction') as NodeListOf<HTMLImageElement>;
```

Comment:
Is there a reason you wouldn't just embed this directly into the az_data_welcome_page directly?

Reply:
No reason other than i was originally having issues with finding the relative path from the az_data_welcome_page to the media folder. It's fixed now.